distributed/tests/test_joblib.py

```diff
@@ -140,3 +135,46 @@ def test_joblib_scatter(loop, joblib):
     assert x.count == 1
     assert y.count == 1
     assert z.count == 4
+
+
+@pytest.mark.parametrize('joblib', joblibs)
+def test_nested_backend_context_manager(loop, joblib):
+    if joblib is None or LooseVersion(joblib.__version__) <= "0.11.0":
+        pytest.skip("Joblib >= 0.11.1 required.")
+    Parallel = joblib.Parallel
+    delayed = joblib.delayed
+
+    def get_nested_pids():
+        return Parallel(n_jobs=2)(delayed(os.getpid)() for _ in range(2))
```

Comment:
It would be more interesting to have:

```python
def get_nested_pids():
    pids = set(Parallel(n_jobs=2)(delayed(os.getpid)() for _ in range(2)))
    return pids.union(Parallel(n_jobs=2)(delayed(os.getpid)() for _ in range(2)))
```

Reply:
This was a good test.  It failed before and passes with the most recent commit.